### PR TITLE
issue-1146 - Update layout of Meeting Minutes page

### PR DESCRIPTION
**Pull request by ericpoirier**
#1146 

Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

**Comment by ericpoirier** [comment]
We have +1 from Shabnam.

File: less/components/blocks.less [hunk 1]
```diff
@@ -25,4 +50,30 @@
     &:hover {
         color: darken(@bordered-box-link-color, 15%);
     }
+}
+
+.flag-box {
```
Comment (autumnfound) [inline less/components/blocks.less]: Couldn't we have used what we have for cards here? https://eclipsefdn-hugo-solstice-theme.netlify.app/components/cards/ all we need to do to align it properly is add a class that unsets left position, and sets right using whats in the left position

Comment (ericpoirier) [reply less/components/blocks.less]: Didn't know that existed, I'll using that instead. Thx.

**Review by chrisguindon — CHANGES_REQUESTED** [review]
We need to add a redirect rule to avoid breaking any existing link: https://github.com/jakartaee/jakarta.ee/blob/src/static/_redirects

**Comment by ericpoirier** [comment]
> We need to add a redirect rule to avoid breaking any existing link: https://github.com/jakartaee/jakarta.ee/blob/src/static/_redirects

I added the redirect to my PR.

**Review by chrisguindon — APPROVED** [review]

**Pull request merged**